src/components/editorWelcome/index.tsx

```diff
@@ -0,0 +1,141 @@
+  textPrimary,
+  textSecondary
+} from "../../constants";
+
+const StyledEditorContainer: AnyStyledComponent = styled.div`
+  width: 100vw;
+  display: flex;
+  flex-direction: row;
+  justify-items: stretch;
+  background: ${background};
+  overflow: auto;
+`;
+
+const StyledWelcomeContainer: AnyStyledComponent = styled.div`
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+
+  h2 {
+    color: ${textPrimary};
+    font-size: 24px;
+    margin: 0 0 0.5rem 0;
+  }
+`;
+
+const StyledWelcomeHeader: AnyStyledComponent = styled.header`
+  padding: 1rem;
+  border: 1px solid ${grey};
+  border-left: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+const StyledWelcomeSection: AnyStyledComponent = styled.main`
+  padding: 4rem 4rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: stretch;
+
+  h1 {
+    color: ${textPrimary};
+    font-size: 48px;
+    margin-bottom: 1rem;
+  }
+
+  p {
+    color: ${textSecondary};
+  }
+`;
+
+const StyledWelcomeInfo: AnyStyledComponent = styled.div`
+  flex: 2;
+  padding-right: 2rem;
+`;
+
+const StyledLink: AnyStyledComponent = styled.a`
+  color: ${accent};
+
+  &:hover {
+    cursor: pointer;
+    text-decoration: underline;
+  }
+`;
+
+const StyledCard: AnyStyledComponent = styled.div`
+  max-width: 800px;
+  margin: 2rem 4rem 2rem 0;
+  padding: 1rem;
+  background: ${codeColor};
+  border-radius: 2px;
+`;
+
+const StyledWelcomeSidebar: AnyStyledComponent = styled.div`
+  flex: 1;
+`;
+
+const EditorWelcome: React.FC = () => {
+  const [editorState, setEditorState] = useGlobal("editorState");
+
+  const createNewTutorial = (): void => {
+    setEditorState({ ...editorState, selectedTutorial: "New Tutorial" });
+  };
+
+  return (
+    <StyledEditorContainer>
+      <Navbar />
+      <StyledWelcomeContainer>
+        <StyledWelcomeHeader>
+          <h2>Tutorial Editor</h2>
+        </StyledWelcomeHeader>
+        <StyledWelcomeSection>
+          <StyledWelcomeInfo>
+            <h1>Propguard Editor</h1>
+            <p>Pass the torch to the next generation by writing tutorials.</p>
+            <StyledCard>
+              <h2>Modify</h2>
+              <p>Modification features are coming soon!</p>
+            </StyledCard>
+            <StyledCard>
+              <h2>Create</h2>
+              <StyledLink onClick={createNewTutorial}>
+                Create a brand new tutorial from scratch
+              </StyledLink>
+            </StyledCard>
+          </StyledWelcomeInfo>
+          <StyledWelcomeSidebar>
+            <h2>Need to update propguard?</h2>
+            <p>
+              <a
+                href="https://github.com/MissouriMRR/propguard/blob/master/contributing-tutorials.md"
+                target="blank"
+              >
+                Click here
+              </a>{" "}
+              for instructions on updating the live Propguard instance or
+              contact one of the maintainers below.
+            </p>
+            <ul>
```

Comment:
I'd make these names a different color than black because they blend into the background. Maybe make them the same color as the text above it or something else a bit lighter.

<img width="440" alt="Screen Shot 2021-03-30 at 8 18 17 PM" src="https://user-images.githubusercontent.com/46657460/113076627-139ad180-9195-11eb-9a96-502f96530191.png">